Patch configure to look for FTS instead of blindly forcing the linkage.

Upstream-Status: Pending
Signed-off-by: Ross Burton <ross.burton@arm.com>

diff --git a/configure.ac b/configure.ac
index cbe5cef..247ea26 100644
--- a/configure.ac
+++ b/configure.ac
@@ -174,6 +174,10 @@ AC_FUNC_REALLOC
 AC_FUNC_STAT
 AC_CHECK_FUNCS([getmntent hasmntopt memset mkdir rmdir strdup])
 
+AC_SEARCH_LIBS([fts_open], [fts], [], [
+  AC_MSG_ERROR([unable to find the fts_open() function])
+])
+
 if test x$with_pam = xtrue; then
 	AC_CHECK_LIB(
 		[pam],
